guarddog/analyzer/sourcecode/bidirectional-characters.yml

```diff
@@ -0,0 +1,20 @@
+rules:
+  - id: bidirectional-characters
+    message: This package contains contains bidirectional (bidi) characters
+    metadata:
+      description: Identify when a package contains contains bidirectional (bidi) characters. See more at https://trojansource.codes/
```

Comment:
`Identify when a package contains contains bidirectional` there are 2x contains -> also in README and bidirectional-characters.py